#Expyriment 1.0.0 (Python 3.10.5), .xpe-file, coding: UTF-8
#date: Sat Oct 11 2025 15:05:07
#sha1: 8b60c3
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Linux', node='eoghan-Precision-7530', release='6.8.0-85-generic', version='#85-Ubuntu SMP PREEMPT_DYNAMIC Thu Sep 18 15:26:59 UTC 2025', machine='x86_64')
Time,Type,Event,Value,Detail,Detail2
2274,design,log,
#design: #exp: Blindspot
#design: #dvn: block nb,trial nb,type,word,color,RT,accuracy
#design: block_cnt,block_id,trial_cnt,trial_id
2274,design,logged,
2274,Experiment,started
3119,Keyboard,received,13,wait
3135,Stimulus,presented,0
4931,Keyboard,received,13,wait
6470,Keyboard,received,32,wait
6472,Stimulus,presented,1
6483,stderr,received,'Traceback (most recent call last):'
6483,stderr,received,'  File "/home/eoghan/Documents/cog_sup/experimental_app/Assigment/Week-5/Exercises/stroop_dog.py", line 59, in <module>'
6483,stderr,received,'    run_block()'
6483,stderr,received,'  File "/home/eoghan/Documents/cog_sup/experimental_app/Assigment/Week-5/Exercises/stroop_dog.py", line 52, in run_block'
6483,stderr,received,'    block+=1'
6483,stderr,received,"UnboundLocalError: local variable 'block' referenced before assignment"
